lib/routes/account.js

```diff
@@ -442,15 +452,19 @@ module.exports = function (
           }
         }
 
-        function sendNewDeviceLoginNotification () {
-          if (config.newLoginNotificationEnabled && wantsKeys(request)) {
+        function sendVerifyLoginEmail() {
+          if (wantsKeys(request)) {
```

Comment:
I worry it would take us too far outside the realm of "expected" UX for users logging into sites on the web.  Signing into a browser is slightly different so I'm hopeful we can get away with a slightly-different-than-expected flow there.